### PR TITLE
Additional settings for postsubmit values

### PR DESCRIPTION
**Pull request by utka**

### CODE REVIEW

**Comment by utka** [comment]
Without this limit the request is bigger then default limit. And set up fails with spec.template.spec.containers[1].resources.requests: Invalid value: "256Mi": must be less than or equal to memory limit

File: perf/istio-install/values-istio-postsubmit.yaml [hunk 1]
```diff
@@ -78,6 +80,11 @@ gateways:
     - port: 5204
       name: tcp-iperf-tls
 
+policy:
```
Comment (howardjohn) [inline perf/istio-install/values-istio-postsubmit.yaml]: doesn't this need to be mixer.policy?

**Comment by howardjohn** [comment]
Ok makes sense. I think the policy one is wrong though

File: perf/istio-install/values-istio-postsubmit.yaml (hunk 1 above)
Comment (utka) [reply perf/istio-install/values-istio-postsubmit.yaml]: No, it is for  istio-policy.

Comment (howardjohn) [reply perf/istio-install/values-istio-postsubmit.yaml]: https://github.com/istio/istio/blob/a952305f6c8489a90515a11bc4ac10b83ced39d2/install/kubernetes/helm/istio/values.yaml#L35-L38
policy is always nested under mixer I think

Comment (linsun) [reply perf/istio-install/values-istio-postsubmit.yaml]: agreed policy should be under mixer.

**Comment by utka** [comment]
updated

**Pull request merged**